db/models/contribution.js

```
@@ -0,0 +1,65 @@
+const Joi = require('joi');
+const db = require('../db');
+const _ = require('lodash');
+
+const schema = Joi.object({
+    user: Joi.number().integer().required(),
+    repository: Joi.number().integer().required(),
+    line_count: Joi.number().integer().required(),
+});
+
+async function insert(data) {
+    try {
+        Joi.assert(data, Repository);
+        const response = await db('contribution').insert(data);
+        return response;
+    } catch (err) {
+        return err;
```

Please don't return an error, just let it be thrown!